test/any-test.js

```diff
@@ -28,11 +28,11 @@ describe('any', () => {
 
 	it('should resolve if at least one input resolves', () => {
 		const s = arrayIterable([reject(1), reject(2), resolve(3)])
-		return any(s).then(x => assert.equal(x, 3))
+		return any(s).then(eq(3))
 	})
 
 	it('should reject if all inputs reject', () => {
 		const s = arrayIterable([1, 2, 3].map(reject))
-		return any(s).catch(() => assert(true))
+		return any(s).then(fail, e => assert(e instanceof Error))
```

Comment:
🐶 😂 Yeah, it'd yield a false positive if the promise returned by any(s) had fulfilled instead of rejecting.